pkgs/applications/misc/hellwal/default.nix

```
@@ -0,0 +1,43 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  makeWrapper,
+}:
+
+let
+  pkgs = import <nixpkgs> {};
+in
+pkgs.stdenv.mkDerivation rec {
+  pname = "hellwal";
+  version = "1.0.2";
+
+  src = pkgs.fetchurl {
+    url = "https://github.com/danihek/hellwal/archive/v1.0.2.tar.gz";
+    sha256 = "sha256-7/ldegQD/AbVeRfktNTb+Jem8DABBFks+VvntUyDvJA=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp hellwal $out/bin/
+
+    mkdir -p $out/share/docs/hellwal/templates
+    cp -r templates/* $out/share/docs/hellwal/templates
+
+    mkdir -p $out/share/docs/hellwal/themes
+    cp -r themes/* $out/share/docs/hellwal/themes/
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/danihek/hellwal";
+    description = "Fast, Extensible Color Palette Generator";
```

See pkgs/README.md for description wording guidelines
```suggestion
    description = "Fast, extensible color palette generator";
```